### PR TITLE
Add configurable position

### PR DESCRIPTION
**Pull request by gravitylow**
Added the option to have the action sheet drop down from the top.

Demo

Thanks for the great lib!

### CODE REVIEW

File: CustomizableActionSheet/CustomizableActionSheet.swift [hunk 1]
```diff
@@ -182,17 +188,26 @@ public class CustomizableActionSheet: NSObject {
         }
       }
     }
+    let positionX: CGFloat = 0
+    let positionY: CGFloat!
+    let moveY: CGFloat!
```
Comment (beryu) [inline CustomizableActionSheet/CustomizableActionSheet.swift]: Do not use `!` .
Please set default value.

File: CustomizableActionSheet/CustomizableActionSheet.swift [hunk 2]
```diff
@@ -212,7 +227,12 @@ public class CustomizableActionSheet: NSObject {
 
     // Hide animation
     self.maskView.alpha = 1
-    let moveY = targetView.bounds.height - self.itemContainerView.frame.origin.y
+    let moveY: CGFloat!
```
Comment (beryu) [inline CustomizableActionSheet/CustomizableActionSheet.swift]: `!` is not necessary in here.
Please remove it.

**Review by beryu — CHANGES_REQUESTED** [review]
This is very interested feature!
Please review my comments.

**Comment by gravitylow** [comment]
Thanks for your comments! The changes have been made in 778952b -- please let me know if you'd like the commits squashed.

**Review by beryu — APPROVED** [review]
Thanks fixing!

**Pull request merged**